buffer/src/main/java/io/netty/buffer/CompositeByteBuf.java

```diff
@@ -648,6 +660,20 @@ protected int _getUnsignedMedium(int index) {
     }
 
     @Override
+    protected int _getUnsignedMediumLE(int index) {
+        Component c = findComponent(index);
+        if (index + 3 <= c.endOffset) {
+            return c.buf.getUnsignedMediumLE(index - c.offset);
+        } else if (order() == ByteOrder.BIG_ENDIAN) {
+            return _getShortLE(index) & 0xFFFF |
+                   (_getByte(index + 2) & 0xFF) << 16;
+        } else {
+            return (_getShortLE(index) & 0xffff) << 8 |
```

Comment:
nit: consider putting on 1 line. This will apply to all lines in this file (which don't exceed the 120 character limit).